packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -30,6 +33,53 @@ exports.sourceNodes = async (
     disallowedLinkTypes,
   } = pluginOptions
   const { createNode } = actions
+
+  if (webhookBody && Object.keys(webhookBody).length) {
+    const changesActivity = reporter.activityTimer(
+      `loading Drupal content changes`,
+      {
+        parentSpan,
+      }
+    )
+    changesActivity.start()
+
+    const { secret, action, id, data } = webhookBody
+    if (pluginOptions.secret && pluginOptions.secret !== secret) {
+      return reporter.warn(
+        `The secret in this request did not match your plugin options secret.`
+      )
+    }
+    if (action === `delete`) {
+      actions.deleteNode({ node: getNode(createNodeId(id)) })
+      return reporter.log(`Deleted node: ${id}`)
+    }
+
+    let nodesToUpdate = data
+    if (!(data instanceof Array)) {
+      nodesToUpdate = [data]
+    }
+
+    for (const nodeToUpdate of nodesToUpdate) {
+      await handleWebhookUpdate(
+        {
+          nodeToUpdate,
+          actions,
+          cache,
+          createNodeId,
+          createContentDigest,
+          getCache,
+          getNode,
+          reporter,
+          store,
+        },
+        pluginOptions
+      )
+    }
+
+    changesActivity.end()
+    return reporter.info(`Processed Drupal content changes`)
```

Comment:
This might've happened as a result of Prettier but we probably want to early return but _not_ return the `reporter.info` call

Reply:
Good call. However, if I try an empty return it seems the linter doesn't like that and errors out (`Async arrow function expected a return value`). Any recommendations for what should be returned instead?

Reply:
Disregard, I figured it out.